interface-definitions/interfaces-tunnel.xml.in

```diff
@@ -283,6 +291,70 @@
                   </leafNode>
                 </children>
               </node>
+              <node name="erspan">
```

Comment:
You open up a new node `erspan` for every possible tunnel, this makes not much sense. Please move this under `set interfaces tunnel tun0 parameters`.